plugins/terraform/resource_load_balancer.go

```diff
@@ -112,7 +112,7 @@ func resourceLayer0LoadBalancerCreate(d *schema.ResourceData, meta interface{})
 	ports := expandPorts(d.Get("port").(*schema.Set).List())
 	healthCheck := expandHealthCheck(d.Get("health_check"))
 
-	if !healthCheck {
+	if healthCheck == (models.HealthCheck{}) {
```

Comment:
This doesn't seem very canonical - could you update `expandHealthCheck` to return  a `*models.HealthCheck` and check `if healthcheck == nil` here?